nextflow.config

```diff
@@ -147,16 +148,17 @@ profiles {
     }
     docker {
         docker.enabled         = true
-        docker.userEmulation   = true
         conda.enabled          = false
         singularity.enabled    = false
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
         apptainer.enabled      = false
+        // Disabled this to deal with libpysal temporary directory issue
+        // docker.runOptions      = '-u $(id -u):$(id -g)'
```

Comment:
Might be able to use this approach for a better fix (likely coming in a template release next week-ish, if it works as we think it will)

https://github.com/nf-core/tools/pull/2673

Reply:
That is good to know, thanks.